.gitignore

```diff
@@ -34,3 +34,6 @@ Thumbs.db
 
 # Files that can be automatically downloaded that we don't want to ship with our builds
 /src/extensibility/node/node_modules/request/tests/
+
+# .disabled should be ignored, it could be used to disable default extensions
+.disabled
```

Comment:
I think it's an overkill. It clearly says default extensions there's nothing to misunderstand.